package.json

```diff
@@ -35,13 +36,15 @@
     "eslint": "^2.1.0",
     "eslint-plugin-json": "^1.2.0",
     "eslint-plugin-mocha": "^2.0.0",
-    "eslint-plugin-react": "^3.16.1",
+    "eslint-plugin-react": "^4.1.0",
```

Comment:
@gingur This was added in #9 because of the differences between `npm@2` and `npm@3` by @mpaulucci and @gshively11. It allows us to have the desired package setup for each version of `npm`. It seems strange, but it doesn't hurt anything.